benchmarks/distributed/rpc/parameter_server/servers/AverageParameterServer.py

```diff
@@ -0,0 +1,98 @@
+import threading
+
+import torch
+import torch.distributed.rpc as rpc
+
+from .AverageParameterServerBase import AverageParameterServerBase
+from .ParameterServerBase import ParameterServerBase
+
+
+class AverageParameterServer(AverageParameterServerBase):
+
+    def __init__(
+        self,
+        rank,
+        trainer_count,
+        lc,
+        use_cuda_rpc
+    ):
+        super().__init__(rank)
+
+        self.lock = threading.Lock()
```

Comment:
pull request to fix issue with instance lock https://github.com/pytorch/pytorch/pull/57943

Reply:
No need to change anything, just FYI. As a temporary solution, IIUC, we can add a `__getstates__` to tell avoid pickling RRefs. 

```python
def __getstates__():
    return {}
```